dvc.yaml

```diff
@@ -1,3 +1,6 @@
+vars:
+  - version: 0.2.0 
```

Comment:
Nice. Does it mean that the package version will be the same as the models version ?

Reply:
Yes, I am looking into a way to pick up the version from the package as well